vulkaninfo/vulkaninfo.cpp

```diff
@@ -619,8 +619,8 @@ void print_usage(const char *argv0) {
     std::cout << "--html                Produce an html version of vulkaninfo output, saved as\n";
     std::cout << "                      \"vulkaninfo.html\" in the directory in which the command is\n";
     std::cout << "                      run.\n";
-    std::cout << "-j, --json            Produce a json version of vulkaninfo output to standard\n";
-    std::cout << "                      output.\n";
+    std::cout << "-j, --json            Produce a json version of vulkaninfo to standard output of the\n";
+    std::cout << "                      first gpu in the system following the dev-sim schema.\n";
```

Comment:
Suggest "conforming to the DevSim schema"

Reply:
Thanks, that is much clearer.